src/components/Search/SearchComponent.tsx

```diff
@@ -137,12 +143,19 @@ function SearchComponent({
           )}
         </Grid>
       </Grid>
-      {checkedRevisionsList.length > 0 && (
+
+      {(checkedRevisionsList.length > 0 ||
+        (selectedRevisions && selectedRevisions.length > 1)) && (
         <Grid>
           <SelectedRevisions
             searchType={searchType}
             mode={mode}
             isWarning={isWarning}
+            selectedRevisions={
+              searchType == 'base'
+                ? selectedRevisions.filter((_, index) => index === 0)
```

Comment:
What do you think of:
```suggestion
                ? [selectedRevisions[0]]
```
This is O(0) while your filter operation is O(n).

It's probably not a big deal here because n is low, but still it's also simpler.

Reply:
Oh I did that first but typescript was unhappy about it so I had to filter the list.  Yes, stupid. I know.

Reply:
Scratch that. I'm going to handle this in the reducer to just get base and new revisions easily. Also, I forgot to put them in an array. -_- Hopefully my brain will work better next sprint.